### PR TITLE
Mcs 521 obstructed movement

### PR DESCRIPTION
**Pull request by kdrumm**
Reduced skin width
Applied scale to determining valid movement
Fixed platform dropping adjustment code.

### CODE REVIEW

File: unity/Assets/Scripts/BaseFPSAgentController.cs [hunk 1]
```diff
@@ -2965,33 +2965,46 @@ public void InitialRandomSpawn(ServerAction action)
...
             RaycastHit hit;
             LayerMask layerMask2 = ~(1 << 10);
 
-            if (Physics.SphereCast(point1, cc.radius, Vector3.down, out hit, Mathf.Infinity, layerMask2))
-            {
+            float adjustedRadius = cc.radius * Mathf.Max(cc.transform.localScale.x, cc.transform.localScale.z);
+            if (Physics.SphereCast(point1, adjustedRadius, Vector3.down, out hit, Mathf.Infinity, layerMask2)) {
                 float raiseMinHeightAboveGround = 0.01f;
-                point2.y = hit.point.y + radius + raiseMinHeightAboveGround;
+                point2.y = hit.point.y + adjustedRadiusAndSkin + raiseMinHeightAboveGround;
```
Comment (ThomasSchellenbergNextCentury) [inline unity/Assets/Scripts/BaseFPSAgentController.cs:2979]: We have a `Changelog` section in the README of this repo in which we (try to) keep notes on all of the changes we make to the original AI2-THOR scripts, so we can reference them in future merges. Would you mind adding a line with a brief summary of your change to `capsuleCastAllForAgent`?

Comment (kdrumm) [reply unity/Assets/Scripts/BaseFPSAgentController.cs:2979]: Sure, ill do that now.

**Comment by ThomasSchellenbergNextCentury** [comment]
To repeat our discussion on Teams:
- Remove the call to `snapAgentToGrid`
- Change the FPSController's CapsuleCollider's radius to match its model's radius.

**Comment by kdrumm** [comment]
Integration tests updated due to the collider radius change.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**